metadata/app.json

```diff
@@ -19,8 +19,8 @@
 		},
 		"en-US": {
 			"label": "Webhooks",
-			"description": "Learn about Kazoo webhooks, create new webhooks, manage and debug existing webhooks.",
-			"extended_description": "Webhooks enables developers to build rigorous integrations that rely on real time, non-critical notifications from KAZOO. Webhooks sends a HTTP, GET, or POST notification to a URL defined by you when certain events occur on a system",
+			"description": "WebHooks is your portal to learn about KAZOO webhooks, create new webhooks, as well as manage and debug existing webhooks.",
```

Comment:
Is it Web`H`ooks or Web`h`ooks? In any case, would be nice if the name was normalized across all copies.